pymc/distributions/multivariate.py

```diff
@@ -507,13 +507,11 @@ class Multinomial(Discrete):
 
     Parameters
     ----------
-    n: int or array
-        Number of trials (n > 0). If n is an array its shape must be (N,) with
-        N = p.shape[0]
-    p: one- or two-dimensional array
-        Probability of each one of the different outcomes. Elements must
-        be non-negative and sum to 1 along the last axis. They will be
-        automatically rescaled otherwise.
+    n: int
```

Comment:
But `n`can be a vector too, can't it? E.g if the number of trials vary by observation

Reply:
Yeah n can be a vector or a matrix, anything as long as dimensions broadcast properly. I put the int to indicate the base case, but perhaps the best is to remove any information of shape and only mention the meaning of the parameters?

Reply:
Why not `int, vector, matrix` ? That's more explicit